packages/MarkdownEditor-Tests.package/MarkdownInlineTextStylerTest.class/instance/testInterpretTokensCutDelimiterClosesTwice.st

```diff
@@ -1,17 +1,20 @@
 tests - interpretation
 testInterpretTokensCutDelimiterClosesTwice
 
+	| firstAttribute secondAttribute |
 	inlineTextStyler 
 		tokenize: '*a**b***';
 		interpretTokens.
 	
 	self assert: 2 equals: inlineTextStyler attributes size.
 	self assert: 6 equals: inlineTextStyler tokens size.
 	
-	self assert: (inlineTextStyler attributes first isKindOf: MarkdownAttributeBold).
-	self assert: 3 equals: inlineTextStyler attributes first startPosition.
-	self assert: 7 equals: inlineTextStyler attributes first endPosition.
+	firstAttribute := inlineTextStyler attributes first.
+	self assert: MarkdownAttribute bold textAttribute equals: firstAttribute textAttribute.
```

Comment:
Actually, we want to test the attribute / emphasis structure similar to the block structure here. I would propose to generalize this a bit, like:

```smalltalk
self
    assert: {MarkdownAttribute bold setFrom: 3 to: 7. MarkdownAttribute italic setFrom: 1 to: 8}
    areEmphasesIn: inlineTextStyler attributes
```
I know that this proposal must be modified because `setFrom:to:` is not quite correct in this context and we need to implement the comparing logic in a new utility, but I think that a similar approach would greatly improve the tests.